sessionctx/variable/varsutil.go

```diff
@@ -687,26 +687,28 @@ func ValidateSetSystemVar(vars *SessionVars, name string, value string) (string,
 		case strings.EqualFold(value, "OFF") || value == "0":
 			return "0", nil
 		case value == "":
-			return "", nil
+			if scope == ScopeSession {
+				return "", nil
+			}
 		}
 		return value, ErrWrongValueForVar.GenWithStackByArgs(name, value)
 	case TiDBStmtSummaryRefreshInterval:
-		if value == "" {
+		if value == "" && scope == ScopeSession {
```

Comment:
why user can't set value = "" globally?

Reply:
I described the problem in the PR description.